### ISSUE TITLE
[BUG] <title>Is there any restriction on images or datasets for fine-tuning vision encoder or llm of MiniCPM-v2.6?

### ISSUE THREAD

**Issue by Wytheyayaya** [OpenBMB/MiniCPM-o#683]
### 是否已有关于该错误的issue或讨论？ | Is there an existing issue / discussion for this?

- [X] 我已经搜索过已有的issues和讨论 | I have searched the existing issues / discussions

### 该问题是否在FAQ中有解答？ | Is there an existing answer for this in FAQ?

- [X] 我已经搜索过FAQ | I have searched FAQ

### 当前行为 | Current Behavior

Are there any recommended images or datasets for fine-tuning vision encoder or llm? I just pick a random image online but loss cannot decrease.

### 运行环境 | Environment

```Markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA (`python -c 'import torch; print(torch.version.cuda)'`):
```

**Comment by LDLINGLINGLING** [comment]
hello，I think the dataset should align for the image and text,if the image and the text in dataset ，If the images and text in the dataset describe the same thing and are highly consistent, the loss will be smaller.
